nrf_802154/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,28 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nRF Connect SDK 1.7.0 - nRF 802.15.4 Radio Driver
+*************************************************
+
+Added
+=====
+
+* Zephyr temperature platform is now used for RSSI correction.
+* Support for the coexistence feature from MPSL.
+* Support for nRF21540 FEM GPIO interface on nRF53 Series.
+
+Notable Changes
+===============
+
+* The 802.15.4 Radio Driver Transmit API has been modified to allow specifying whether to encrypt and/or inject dynamic data into the outgoing frame (via nrf_802154_transmitted_frame_props_t type).
+
+Bug fixes
+=========
+
+* Allow transmission of frames with invalid Auxiliary Security Header in case of NRF_802154_ENCRYPTION=n. (KRKNWK-11218)
```

Comment:
```suggestion
* Fixed an issue where it would not be possible to transmit frames with invalid Auxiliary Security Header if :kconfig:`CONFIG_NRF_802154_ENCRYPTION` was set to ``n``. (KRKNWK-11218)